internal/http/middleware.go

```diff
@@ -4,21 +4,62 @@ import (
 	"context"
 	"fmt"
 	"net/http"
+	"regexp"
 	"strings"
+	"time"
 
-	goboilerplate "github.com/kurio/boilerplate-go"
-
+	"github.com/DataDog/datadog-go/statsd"
 	"github.com/labstack/echo"
 	"github.com/pkg/errors"
-
 	log "github.com/sirupsen/logrus"
+
+	goboilerplate "github.com/kurio/boilerplate-go"
 )
 
+// ResponseTimeMiddleware is used to send response_time metrics to statsd.
+func ResponseTimeMiddleware(statsdClient *statsd.Client) echo.MiddlewareFunc {
+	return func(next echo.HandlerFunc) echo.HandlerFunc {
+		return func(c echo.Context) error {
+			startTime := time.Now()
+
+			err := next(c)
+			if err != nil {
+				return err
+			}
+
+			path := c.Request().URL.Path
+
+			metricName := getMetricName(c.Request().Method + " " + path)
+			if metricName == "" {
+				return nil
+			}
+
+			responseTime := time.Since(startTime)
+
+			tags := []string{
+				"service:boilerplate-go",
```

Comment:
better use `goboilerplate` for easier replacement on CLI app
```suggestion
				"service:goboilerplate",
```